src/main/java/net/testudobank/MvcController.java

```diff
@@ -46,7 +49,7 @@
   public static String TRANSACTION_HISTORY_CRYPTO_BUY_ACTION = "CryptoBuy";
   public static String CRYPTO_HISTORY_SELL_ACTION = "Sell";
   public static String CRYPTO_HISTORY_BUY_ACTION = "Buy";
-  public static String CRYPTO_NAME = "ETH";
+  public static Set<String> SUPPORTED_CRYPTOCURRENCIES = new HashSet<>(Arrays.asList("ETH", "SOL"));
```

Comment:
I would consider saying `Set<String> SUPPORTED_CRYPTOCURRENCIES = Set.of("ETH","SOL");` here (introduced in Java 9)